NomNomAPI/Services/FoodItemService/FoodItemService.cs

```diff
@@ -53,6 +66,34 @@ public async Task<IEnumerable<FoodItem>> GetAllFoods()
             return foods;
         }
 
+        public async Task<FoodItem?> GetSingleFood(int id)
+        {
+            var food = await _context.foodItems.FindAsync(id);
+            if (food == null)
+                return null;
+            return food;
+        }
+
+        public async Task<List<FoodItem>?> UpdateFood(int id, FoodItem request)
+        {
```

Comment:
This function updates a single item, so it should not return all of them. Usually it either returns the modified object or void.